FlashEnglish/Sources/Views/Home/HomeView.swift

```diff
@@ -12,6 +12,12 @@ struct HomeView: View {
     @EnvironmentObject var quizManager: QuizManager
     @EnvironmentObject var navigationManager: NavigationManager
     @Binding var presentSideMenu: Bool
+
+
+
+
+
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Masaya1582/FlashEnglish/blob/26f8fd3d00e8b28f320403772b345ccefdcdf0b2/FlashEnglish/Sources/Views/Home/HomeView.swift#L20"></span>Limit vertical whitespace to a single empty line; currently 6<br />
<code>vertical_whitespace</code> <code>HomeView.swift:20</code></td>
    </tr>
  </tbody>
</table>